src/app/shared/dialogs/dialogs-list.component.html

```diff
@@ -30,6 +30,11 @@
   [pageSizeOptions]="[5, 10, 20, 50, 100, 200]">
 </mat-paginator>
 <mat-dialog-actions>
+  <button mat-raised-button
```

Comment:
This should not be a button because there is no action when clicked.  Should just be text